crates/pop-cli/src/commands/install/mod.rs

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: GPL-3.0
+use crate::style::{style, Theme};
+use anyhow::Context;
+use clap::Args;
+use cliclack::{clear_screen, confirm, intro, log, outro, set_theme};
+use duct::cmd;
+use os_info::Type;
+use strum::Display;
+use tokio::{fs, process::Command};
+
+#[derive(Display)]
+pub enum Dependencies {
+	#[strum(serialize = "build-essential")]
+	BuildEssential,
+	#[strum(serialize = "clang")]
+	Clang,
+	#[strum(serialize = "clang-devel")]
+	ClangDevel,
+	#[strum(serialize = "cmake")]
+	Cmake,
+	#[strum(serialize = "curl")]
+	Curl,
+	#[strum(serialize = "gcc")]
+	Gcc,
+	#[strum(serialize = "git")]
+	Git,
+	#[strum(serialize = "homebrew")]
+	Homebrew,
+	#[strum(serialize = "libclang-dev")]
+	LibClang,
+	#[strum(serialize = "libssl-dev")]
+	Libssl,
+	#[strum(serialize = "make")]
+	Make,
+	#[strum(serialize = "openssl")]
+	Openssl,
+	#[strum(serialize = "openssl-devel")]
+	OpenSslDevel,
+	#[strum(serialize = "pkg-config")]
+	PkgConfig,
+	#[strum(serialize = "protobuf")]
+	Protobuf,
+	#[strum(serialize = "protobuf-compiler")]
+	ProtobufCompiler,
+	#[strum(serialize = "rustup")]
+	Rustup,
+}
+
+#[derive(Args)]
+#[command(args_conflicts_with_subcommands = true)]
+/// Setup user environment for development
+pub(crate) struct InstallArgs {
+	/// Automatically install all dependencies required without prompting for confirmation.
+	#[clap(short('y'), long)]
+	skip_confirm: bool,
+}
+
+impl InstallArgs {
+	pub(crate) async fn execute(self) -> anyhow::Result<()> {
+		clear_screen()?;
+		set_theme(Theme);
+		intro(format!(
+			"{}: Install dependencies for development",
+			style(" Pop CLI ").black().on_magenta()
+		))?;
+		if cfg!(target_os = "macos") {
+			log::info("ℹ️ Mac OS (Darwin) detected.")?;
+			install_mac(self.skip_confirm).await?;
+		} else if cfg!(target_os = "linux") {
+			match os_info::get().os_type() {
+				Type::Arch => {
+					log::info("ℹ️ Arch Linux detected.")?;
+					install_arch(self.skip_confirm).await?;
+				},
+				Type::Debian => {
+					log::info("ℹ️ Debian Linux detected.")?;
+					install_debian(self.skip_confirm).await?;
+				},
+				Type::Redhat => {
+					log::info("ℹ️ Redhat Linux detected.")?;
+					install_redhat(self.skip_confirm).await?;
+				},
+				Type::Ubuntu => {
+					log::info("ℹ️ Ubuntu detected.")?;
+					install_ubuntu(self.skip_confirm).await?;
+				},
+				_ => return not_supported_message(),
+			}
+		} else {
+			return not_supported_message();
+		}
+		install_rustup().await?;
+		outro("✅ Installation complete.")?;
+		Ok(())
+	}
+}
+
+async fn install_mac(skip_confirm: bool) -> anyhow::Result<()> {
+	log::info("More information about the packages to be installed here: https://docs.substrate.io/install/macos/")?;
+	if !skip_confirm {
+		prompt_for_confirmation(&format!(
+			"{}, {}, {}, {} and {}",
+			Dependencies::Homebrew,
```

Comment:
Can you not use `use Dependencies::*` at top of file to avoid this prefix constantly?